README.md

```diff
@@ -83,46 +83,43 @@ All releases on the same Major version will be guaranteed to have backward compa
 
 ## Usage
 
-OSV-Scanner collects a list of dependencies and versions that are used in your project, before matching this list against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api). To build the list of dependencies, you can point OSV-Scanner at your project directory, or manually pass in the path to individual manifest files.
+OSV-scanner parses lockfiles, SBOMs, and git directories to determine your project's open source dependencies. These dependencies are matched against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api) and known vulnerabilities are returned to you in the output. 
 
-### Scan a directory
+### General Use Case: Scanning a Directory
 
-Walks through a list of directories to find:
+```console
+osv-scanner -r /path/to/your/dir
+```
 
-- Lockfiles
-- SBOMs
-- git directories for the latest commit hash
+The above command will find lockfiles, SBOMs, and git directories in your target directory and use them to determine the dependencies to check against the OSV database. It will give you the most complete understanding of the known vulnerabilities in your project. 
```

Comment:
what do you think about this instead?

```suggestion
The above command will find lockfiles, SBOMs, and git directories in your target directory and use them to determine the dependencies to check against the OSV database for any known vulnerabilities.
```

How it's currently written feels a bit stiff and false to me as "most complete understanding" isn't being compared to anything in particular, and while I think the scanner does a good job it definitely doesn't give _the_ most complete understanding possible

Reply:
I was trying to compare scanning a single lockfile compared to scanning the directory. But I think your point is reasonable.